### DISCUSSION TITLE
Agent node disk inexplicably skyrocketed and then disconnected

### DISCUSSION THREAD

**Discussion by DokiDoki1103** [k3s-io/k3s#9775] [Q&A]
**Environmental Info:**
K3s Version: 

k3s version v1.28.7+k3s1 (051b14b2)
go version go1.21.7
Node(s) CPU architecture, OS, and Version: 

Linux VM-20-16-centos 3.10.0-1160.108.1.el7.x86_64 #1 SMP Thu Jan 25 16:17:31 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
Cluster Configuration:

1 servers, 5 agents
**Describe the bug:**

I deployed K3s through the public network。After using it for a period of time, the agent node's disk usage skyrocketed, and then the node crashed。Using the Wireguard native network plugin。
**Steps To Reproduce:**

- Installed K3s: 

**Expected behavior:**

**Actual behavior:**

**Additional context / logs:**

**Comment by brandond** [comment]
> the agent node's disk usage skyrocketed, and then the node crashed

Can you provide logs showing why it crashed?

How much disk does this node have available, and what specifically was consuming space?

K3s will not fill the disk just sitting there on its own, so something else must be going on, but you've provided absolutely no information that might assist in figuring out what that might be.

**Comment by DokiDoki1103** [comment]
<img width="1378" alt="图片" src="https://github.com/k3s-io/k3s/assets/62740231/9b347cbf-0f5b-4863-a33e-6def3e1e1837">
<img width="1362" alt="图片" src="https://github.com/k3s-io/k3s/assets/62740231/9fec85b4-a186-44a7-986a-9f8eb1872ce9">

I don't know what I can offer you, but the cluster I assembled will soon have a few nodes that are not ready. I checked the Tencent Cloud backend and found that the disks are busy

**Comment by DokiDoki1103** [comment]
If the node is not ready, I cannot connect to the SSH server and can only go to the Tencent Cloud backend to restart the server
